core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -188,7 +199,7 @@ object WorkspaceMigrationMonitor {
     } yield ()
 
 
-  final def issueWorkspaceBucketTransferJob: MigrateAction[TransferJob] =
+  final def issueWorkspaceBucketTransferJob: MigrateAction[Unit] =
```

Comment:
Why do we want to change this from a TransferJob to a Unit? My preference is usually more specific return types, rather than less, is there a reason that this needs to return things other than a TransferJob? Does it?

Reply:
Because the return value is not used and the status of the transfer job is tested elsewhere.

Reply:
Maybe its my experiences working in typeless languages, but even if the return type isn't being used, do you think it is more clear to make this return something else then if the output isn't being used? I am personally concerned about future usage and clarity -> I would prefer either we explicitly return something unusable (so that people looking at this function explicitly don't use the returned object), or we clarify the return type so that users know exactly what they are getting and don't end up with accidental type mismatches down the line.

Reply:
I don't know. I guess I'm just viewing each of these steps as just side effects and the migration "pipeline" as a sequencing of these side effects. Personally I feel that unused return values are a bit of a code smell. I'm not sure what you mean by returning something unusable. Could you clarify?

Reply:
Discussed offline.